File: app/src/main/java/com/example/administrator/essim/activities/AboutActivity.java
package com.example.administrator.essim.activities;

import android.content.Context;
import android.os.Bundle;
import android.support.design.widget.FloatingActionButton;
import android.support.v7.app.AppCompatActivity;
import android.view.WindowManager;

import com.example.administrator.essim.R;
import com.sdsmdg.tastytoast.TastyToast;

import java.io.File;
import java.io.IOException;
import java.util.zip.ZipException;


public class AboutActivity extends AppCompatActivity {

    private Context mContext;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN);
        setContentView(R.layout.activity_about);

        mContext = this;
        FloatingActionButton fab = findViewById(R.id.fab_about);
       // fab.setOnClickListener(null);
    }
}

File: app/src/main/java/com/example/administrator/essim/activities/AboutActivity.kt
package com.example.administrator.essim.activities

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.WindowManager

import com.example.administrator.essim.R


class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN)
        setContentView(R.layout.activity_about)
    }
}
